src/ui/FileList/FileList.stories.tsx

```diff
@@ -264,16 +264,15 @@ function Example() {
                           ? Math.floor(Math.random() * 100)
                           : null
                       }
-                      headNumber={i + 1}
-                      baseNumber={i}
+                      headNumber={`${i + 1}`}
+                      baseNumber={`${i}`}
```

Comment:
why do this?

Reply:
`baseNumber` requires a string variable ...

Reply:
ahhh ok and we weren't giving it one before got it